enhancements/scheduling/scheduler-profiles.md

```diff
@@ -0,0 +1,212 @@
+
+### Non-Goals
+
+* fine configuration of individual scheduler plugins
+  * This differs from the current scheduler Policy config, which allows individual control over predicates and priorities
+* default support for custom schedulers or out-of-tree plugins
+
+Note that these non-goals are only for the default cluster scheduler managed by OpenShift, and should not imply any inherent platform restrictions on secondary or other custom schedulers users choose to deploy on their own.
+
+## Proposal
+
+This enhancement proposes a new field in `schedulers.config.openshift.io/v1` named `profile`. This field
+specifies which of the following plugin configurations (as scheduler profiles to enable on the default cluster
+scheduler.
+
+* `LowUtilization`: this will use the default scheduler profile as defined in the kube-scheduler's internal registry.
+This is the equivalent of currently passing no config to the scheduler, and in theory may change if the scheduler's
+list of default ("in-tree") plugins changes.
+  * Currently, this equates to a "Low Utilization" profile, with `NodeResourcesLeastAllocated` enabled
+
+* `HighUtilization`: this will disable `NodeResourcesLeastAllocated` and enable `NodeResourcesMostAllocated` to enable
+bin-packing of workloads onto nodes.
+
+* `NoScoring`: this is a "low latency" profile which strives for the quickest scheduling cycle by disabling all score plugins. 
+Score plugins are inherently non-critical so their exclusion should be safe and provide decreased scheduling time.
+
+### User Stories [optional]
+
+#### Story 1
+
+As a sysadmin, I want my workloads to be packed tightly onto nodes rather than the default behavior of spreading workloads 
+out among nodes. I enable the `HighUtilization` profile.
+
+#### Story 2
+
+As a sysadmin, I want to schedule my workloads as quickly as possible without regard for any soft preference of specific nodes. 
+I enable the `NoScoring` profile to achieve this.
+
+### Implementation Details/Notes/Constraints [optional]
+
+The new field will be set in the operator config:
+```yaml
+kind: Scheduler
+apiVersion: config.openshift.io/v1
+spec:
+  ...
+  profile: Default | HighUtilization | NoScoring | ...
+```
+
+The available values will be constrained to those defined above by API validation, but there is the possibility of adding more 
+options if significant use cases arise for different configurations.
+
+### Risks and Mitigations
+
+* Security and stability risks should be minimal, as this method of configuration is actually more restrictive than what is 
+currently supported for the scheduler
+* However, due to that there is a risk that this regresses from the current depth of configuration supported for customers with 
+very specific needs (based on metrics gathered, there are likely very few clusters that would be impacted in this way)
+* Transition from the old Policy API will provided by supporting the new Profile field alongside the old API (at least until Policy is 
```

Comment:
This makes sense, but in addition we should find the list of customers and try to individually contacting them to see about migration.

Reply:
> This makes sense, but in addition we should find the list of customers and try to individually contacting them to see about migration.

When you discover all of these, please create a private jira card listing everything  we've currently got.  I think we'll have to work our way down the list.

Reply:
Yeah, we reached out to @jeremyeder who wasn't aware of anyone currently requesting this feature, though with it documented there may be some who have done it themselves. I wonder if it would be possible to work with telemetry to correlate the (albeit low) numbers we saw for current Policy config clusters to contact them.